odoo_module_migrate/migration.py

```diff
@@ -120,6 +134,13 @@ def _get_code_from_previous_branch(self, module_name, remote_name):
                 'module': module_name,
             }, path=self._directory_path)
 
+        logger.info("AutoSquash Commit")
+        _execute_shell(
+            "GIT_EDITOR=odoo-module-squasher "
+            "GIT_SEQUENCE_EDITOR=odoo-module-squasher "
+            "git rebase  -i %(target)s" % {"target": target_version}
```

Comment:
Should optional, I think

Reply:
Done, I have added the option --auto-squash